e2e/test-component/scenarios/embedding-sdk/interactive-question-native.cy.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+import { createNativeQuestion, describeEE } from "e2e/support/helpers";
+import {
+  mockAuthProviderAndJwtSignIn,
+  mountInteractiveQuestion,
+  signInAsAdminAndEnableEmbeddingSdk,
+} from "e2e/support/helpers/component-testing-sdk";
+import type { DatasetColumn } from "metabase-types/api";
+
+const { ORDERS, ORDERS_ID } = SAMPLE_DATABASE;
+
+describeEE("scenarios > embedding-sdk > interactive-question > native", () => {
+  beforeEach(() => {
+    signInAsAdminAndEnableEmbeddingSdk();
+  });
+
+  it("supports passing sql parameters to native questions", () => {
+    createNativeQuestion(
+      {
+        native: {
+          query: "SELECT * FROM orders WHERE {{ID}}",
+          "template-tags": {
+            ID: {
+              id: "6b8b10ef-0104-1047-1e1b-2492d5954322",
+              name: "ID",
+              "display-name": "ID",
+              type: "dimension",
+              dimension: ["field", ORDERS.ID, null],
+              "widget-type": "category",
+              default: null,
+            },
+          },
+        },
+      },
+      { wrapId: true },
+    );
+
+    cy.signOut();
+    mockAuthProviderAndJwtSignIn();
```

Comment:
nit: we usually keep the setup in `beforeEach`

Reply:
Updated in https://github.com/metabase/metabase/pull/50728/commits/d291c7fafbbdbb82b7cedc5291d7f4cff2dc4215, thanks!